### PR TITLE
Feat recovery.mode=import

### PR DESCRIPTION
**Pull request by itay-grudev**

### CODE REVIEW

**Review by philippemnoel — APPROVED** [review]

**Comment by philippemnoel** [comment]
This looks super clean!!

File: charts/paradedb/README.md [hunk 1]
```diff
@@ -204,15 +204,15 @@ refer to  the [CloudNativePG Documentation](https://cloudnative-pg.io/documentat
...
-| cluster.monitoring.enabled | bool | `true` | Whether to enable monitoring |
+| cluster.monitoring.enabled | bool | `false` | Whether to enable monitoring |
 | cluster.monitoring.podMonitor.enabled | bool | `true` | Whether to enable the PodMonitor |
 | cluster.monitoring.podMonitor.metricRelabelings | list | `[]` | The list of metric relabelings for the PodMonitor. Applied to samples before ingestion. |
 | cluster.monitoring.podMonitor.relabelings | list | `[]` | The list of relabelings for the PodMonitor. Applied to samples before scraping. |
 | cluster.monitoring.prometheusRule.enabled | bool | `true` | Whether to enable the PrometheusRule automated alerts |
 | cluster.monitoring.prometheusRule.excludeRules | list | `[]` | Exclude specified rules |
 | cluster.postgresGID | int | `-1` | The GID of the postgres user inside the image, defaults to 26 |
 | cluster.postgresUID | int | `-1` | The UID of the postgres user inside the image, defaults to 26 |
-| cluster.postgresql.parameters | object | `{"cron.database_name":"postgres"}` | PostgreSQL configuration options (postgresql.conf) |
```
Comment (philippemnoel) [inline charts/paradedb/README.md:215]: This shouldn't be removed, I think.

File: charts/paradedb/README.md [hunk 2]
```diff
@@ -282,18 +302,18 @@ refer to  the [CloudNativePG Documentation](https://cloudnative-pg.io/documentat
...
+| poolers[].monitoring.enabled                        | bool                                         | `false`                                          | Whether to enable monitoring for the Pooler.                                                                                                                                                                                                                                                                                                                                                                                               |
+| poolers[].monitoring.podMonitor.enabled             | bool                                         | `true`                                           | Create a podMonitor for the Pooler.                                                                                                                                                                                                                                                                                                                                                                                                        |
```
Comment (philippemnoel) [inline charts/paradedb/README.md:316]: Can we not rewrite this? It should be the same

File: charts/paradedb/README.md.gotmpl [hunk 3]
```diff
@@ -41,20 +41,6 @@ helm upgrade --atomic --install cnpg \
...
-```bash
-helm repo add cnpg https://cloudnative-pg.github.io/charts
-helm upgrade --atomic --install cnpg \
---create-namespace \
---namespace cnpg-system \
---set monitoring.podMonitorEnabled=true \
---set monitoring.grafanaDashboard.create=true \
-cnpg/cloudnative-pg
-```
-
```
Comment (philippemnoel) [inline charts/paradedb/README.md.gotmpl:57]: I don't think we should remove this either

**Review by philippemnoel — APPROVED** [review]

File: charts/paradedb/README.md (hunk 2 above)
Comment (itay-grudev) [reply charts/paradedb/README.md:316]: My IDE just re-aligns Markdown tables. That's a symptom of my OCD.

File: charts/paradedb/README.md (hunk 1 above)
Comment (philippemnoel) [reply charts/paradedb/README.md:215]: Ah yes, it is correct

File: charts/paradedb/README.md.gotmpl (hunk 3 above)
Comment (philippemnoel) [reply charts/paradedb/README.md.gotmpl:57]: Ah yes, this was a duplicate -- good catch

**Pull request merged**